helm/steps-to-deploy/READme.md

```diff
@@ -0,0 +1,35 @@
+## Steps to Deploy ERP into K8s Cluster
+
+1. Clone git repository for helm chart: https://github.com/truthfool/openg2p-erp-docker.git
+
+
+2. Create persistent volume claim in cloud using :
+`kubectl apply -f pvc.yaml`
+
+
+3. Deploy odoo and postgresql pod using :
+ `helm install openg2p -f values.yaml bitnami/odoo`
```

Comment:
Is there a particular path this command needs to be executed from or from the base path of cloned repository?

Reply:
i will change the folder.